bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/connection/OneWireConnection.java

```diff
@@ -255,30 +332,30 @@ public static synchronized String readFromOneWire(AbstractOneWireDevicePropertyB
 
     /**
      * Writes String to 1-Wire device property
-     * 
+     *
      * @param pvDevicePropertyPath
      * @param pvValue
      */
     public static synchronized void writeToOneWire(String pvDevicePropertyPath, String pvValue) {
         int lvAttempt = 1;
         while (lvAttempt <= cvRetry) {
             try {
-                logger.debug("trying to write '{}' to '{}', write attempt={}",
+                logger.debug("Trying to write '{}' to '{}', write attempt={}",
                         new Object[] { pvValue, pvDevicePropertyPath, lvAttempt });

```

Comment:
Instead of `new Object []`, just pass the object args, to decrease object instantiation/garbage collection.